shared/applicability/package.yml

```diff
@@ -0,0 +1,12 @@
+name: 'cpe:/a:{arg}:{ver_cpe}'
+title: 'Package {arg} {ver_str} is installed'
+template:
+    name: cond_package
+args:
+    chronytest:
```

Comment:
You should use a templated platform in at least 1 rule. Now you introduce something that isn't used anywhere. If it would be used in some rule, then the new code would be tested by all forms of CI as a part of this CI gating. When it isn't used anywhere, the reviewer tries to try it out but he probably won't catch all quirks of different CI runs on his laptop.